build/rule-generator.js

```diff
@@ -0,0 +1,52 @@
+/* global Promise */
+
+const fs = require('fs');
+const chalk = require('chalk');
+const execa = require('execa');
+
+const { getAnswers } = require('./rule-generator/get-answers');
+const { getFilesMetaData } = require('./rule-generator/get-files-metadata');
+const { createFile, directories } = require('./rule-generator/utils');
+
+const run = async () => {
+	// ensure axe exists, if not build axe, then start the generator
+	const axeExists = fs.existsSync(directories.axePath);
+	if (!axeExists) {
+		console.log(
+			chalk.red.bold(
+				`Axe does not exist. Triggering build using - 'npm run build'. Rule generation will continue after build.`
+			)
+		);
+		await execa.shell('npm run build');
+	}
+
+	// rule-generator banner
+	console.log(chalk.hex('#0077c8')('Axe Rule Generator'));
+
+	// get answers (ask questions)
+	const answers = await getAnswers();
+
+	// get metadata of files to be created
+	const files = getFilesMetaData(answers);
+
+	// create the files
+	if (!files || !files.length) {
+		console.log(chalk.red.bold(`No files to generate.`));
+	}
+
+	try {
+		const result = await Promise.all(files.map(createFile));
+		console.log(
```

Comment:
The only thing I'd add to this is an extra space before "Successfully generated" to give a little more breathing room between sections. Otherwise it works great!